### PR TITLE
[command] fix ck issues

### PR DESCRIPTION
**Pull request by fenrir-z**
# how to reproduce
run command merge, mining, filter, sampling, evaluate and you'll find cks lost

### CODE REVIEW

**Review by yance-dev — DISMISSED** [review]

File: ymir/command/mir/commands/evaluate.py [hunk 1]
```diff
@@ -55,6 +55,8 @@ def run_with_args(work_dir: str, src_revs: str, dst_rev: str, mir_root: str, con
             new_mir_annotation.task_annotations[task_id].CopyFrom(pred_annotations)
             new_mir_annotation.prediction.CopyFrom(pred_annotations)
             new_mir_annotation.ground_truth.CopyFrom(mir_annotations.ground_truth)
+            for asset_id in mir_annotations.image_cks.keys():
```
Comment (phoenix-xhuang) [inline ymir/command/mir/commands/evaluate.py:58]: `new_mir_annotation.image_cks.update(mir_annotations.image_cks)`

File: ymir/command/mir/commands/filter.py [hunk 2]
```diff
@@ -158,6 +158,10 @@ def run_with_args(mir_root: str, in_cis: Optional[str], ex_cis: Optional[str], i
                                          dst_task_annotations=matched_mir_annotations.prediction,
                                          asset_ids=asset_ids_set)
 
+        image_ck_asset_ids = asset_ids_set & set(mir_annotations.image_cks.keys())
```
Comment (phoenix-xhuang) [inline ymir/command/mir/commands/filter.py:161]: `matched_mir_annotations.image_cks.update(mir_annotations.image_cks)`

File: ymir/command/mir/commands/mining.py [hunk 3]
```diff
@@ -254,6 +254,10 @@ def _process_results(mir_root: str, export_out: str, dst_typ_rev_tid: revs_parse
     for asset_id in gt_asset_ids:
         ground_truth.image_annotations[asset_id].CopyFrom(mir_annotations.ground_truth.image_annotations[asset_id])
 
+    image_ck_asset_ids = set(mir_annotations.image_cks.keys() & asset_ids_set)
```
Comment (phoenix-xhuang) [inline ymir/command/mir/commands/mining.py:257]: matched_mir_annotations.image_cks.update({k: mir_annotations.image_cks[k] for k in image_ck_asset_ids})

File: ymir/command/mir/commands/sampling.py [hunk 4]
```diff
@@ -85,6 +85,8 @@ def run_with_args(mir_root: str, work_dir: str, src_revs: str, dst_rev: str, cou
                 mir_annotations.task_annotations[mir_annotations.head_task_id])
             sampled_mir_annotations.prediction.CopyFrom(mir_annotations.prediction)
             sampled_mir_annotations.ground_truth.CopyFrom(mir_annotations.ground_truth)
+            for asset_id in mir_annotations.image_cks.keys():
```
Comment (phoenix-xhuang) [inline ymir/command/mir/commands/sampling.py:88]: sampled_mir_annotations.image_cks.update(xxx)

File: ymir/command/mir/commands/filter.py (hunk 2 above)
Comment (fenrir-z) [reply ymir/command/mir/commands/filter.py:161]: 同上

File: ymir/command/mir/commands/evaluate.py (hunk 1 above)
Comment (fenrir-z) [reply ymir/command/mir/commands/evaluate.py:58]: 类似于 update 的操作，对 protobuf 里面，value 为 Message 的 map 结构都不适用。

File: ymir/command/mir/commands/mining.py (hunk 3 above)
Comment (fenrir-z) [reply ymir/command/mir/commands/mining.py:257]: 同上

File: ymir/command/mir/commands/sampling.py (hunk 4 above)
Comment (fenrir-z) [reply ymir/command/mir/commands/sampling.py:88]: 同上

**Review by phoenix-xhuang — APPROVED** [review]

**Review by IJtLJZ8Rm4Yr — APPROVED** [review]

**Pull request merged**